docs/operational_guide/replication_between_clusters.md

```diff
@@ -0,0 +1,73 @@
+# Replication between clusters (beta)
+
+## Overview
+
+M3DB clusters can be configured to passively replicate data from other clusters. This feature is most commonly used when operators wish to run two (or more) regional clusters that function independently, but that passively replicate data from each other in an eventually consistent manner.
```

Comment:
nit:

> [...] that function independently, but that passively replicate data from each other [...]

technically not a second sentence (so a conjunction breaks the flow):

"[...] that function independently while passively replicating data from the other cluster [...]